ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/StatusResourceTest.java

```diff
@@ -102,4 +105,16 @@ public void testGetStatus() throws Exception {
       assertEquals(expectedCommandStatus, testCommandStatus);
     }
   }
+
+  @Test
+  public void testGetStatusNotFound() throws Exception {
```

Comment:
A side comment out of curiosity.

I wonder if we actually return the full gamut of statuses today. The enum includes things like `QUEUED`, `PARSING`, `RUNNING`, `TERMINATED` etc. Also, if the streams app dies, the state is probably not switched to `TERMINATED` or `ERROR` (which are both valid statuses).

If this is true, then it might be worth filing an issue to improve the functionality of the status endpoint.

Reply:
The endpoint is just writing the status enum and message into the response. So we should be good here.